resources/user-agents/browsers/sundance/sundance-generic.json

```diff
@@ -0,0 +1,44 @@
+{
+  "division": "Sundance Generic",
+  "sortIndex": 1021,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Sundance Generic",
+      "platform": "Windows",
+      "device": "Windows Desktop",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Sundance Generic",
+        "Browser": "Sundance",
+        "Browser_Type": "Browser",
+        "Alpha": "false",
+        "Beta": "false",
+        "isSyndicationReader": "false",
+        "Crawler": "false",
+        "Browser_Maker": "Digola",
+        "Frames": "true",
+        "IFrames": "true",
+        "Tables": "true",
+        "Cookies": "true",
+        "JavaScript": "true",
+        "CssVersion": "3"
+      },
+      "children": [
+        {
+          "match": "Sundance/* (Compatible; Windows; U; en-US)Version/*",
```

Comment:
This one comes from the useragent reported by the Sundance browser after downloading it to a Windows VM I have (http://www.digola.com/sundance.html).

The pattern below I made from an entry in our logs (which a couple of parsers also parse as Sundance). I'm not 100% sure that both of these are the same browser or not. Perhaps early versions of the browser used the UA below, but more recent versions use this one.